### ISSUE TITLE
Update steps in CONTRIBUTING to match steps in CWG2R

### ISSUE THREAD

**Issue by zkamvar** [NESCent/popgenInfo#153]
I've noticed that CONTRIBUTING has 12 steps for the process of submitting a vignette, most of which are specific to first time setup. Should we pare these steps down to the 4 (or six, if starting) steps that are in Contributing With git2R? This would also bring the steps in alignment with [our figure](https://raw.githubusercontent.com/NESCent/popgenInfo/master/images/workflow.png). 

Here's what I'm envisioning:

-----

## Setup for new contributors

If you are a new contributor, you will need to make sure you have git set up and a fork of this repository. Please follow these instructions carefully. If you have already set up your repository, go to the next section. 

1. Set up a GitHub account if you don't already have one, see [here](https://guides.github.com/activities/hello-world/) for a quick tutorial on using GitHub. 
2. Download an appropriate Git client for your computer, see [here](https://help.github.com/articles/set-up-git/), and set it up.
3. Fork the [source repository](http://github.com/NESCent/popgenInfo) on GitHub using the `Fork` button (See [GitHub's tutorial](https://help.github.com/articles/fork-a-repo/)). 
4. On the command line, clone the repository to your machine using    
`git clone https://github.com/YourAccount/popgenInfo.git`    
Note: the https protocol is easier to follow, but it's possible to use ssh as well ([details here](https://help.github.com/articles/fork-a-repo/#step-2-create-a-local-clone-of-your-fork)). 
5. Add the NESCent repository to upstream with     
`git remote add NESCent https://github.com/NESCent/popgenInfo.git`    
to keep your fork up to date with the master repository. [See these instructions for details](https://help.github.com/articles/fork-a-repo/#step-3-configure-git-to-sync-your-fork-with-the-original-spoon-knife-repository)

## Step-by-step process for submission

If you have a vignette you would like to submit, please follow these instructions to submit it for review. 

1. Update your fork
  1. From the command line, move to the **popgenInfo** directory using `cd popgenInfo`
  2. Switch to your master branch with `git checkout master`
  3. Update with    
   ```sh
   git fetch NESCent          # grab all the changes from NESCent
   git merge NEScent/master   # add them to your local fork
   git push                   # push them up to your fork on github
   ```    

2. Create a branch for your changes using `git checkout -b Your_branch_name`.
3. Add your vignette
  1. If you are on windows or macintosh, you can open Rstudio by double-clicking on the **popgenInfo.Rproj** file in your file browser. If you are on linux, you can open it by typing `rstudio popgenInfo.Rproj`.
  2. If you haven't done so, create your vignette from the [`TEMPLATE.Rmd`](https://github.com/nescent/popgenInfo/tree/master/TEMPLATE.Rmd) file and move any associated data to the **data/** directory.
  3. Move your  vignette in `.Rmd` format into the **use/** directory.
  4. Add the file to git using `git add use/yourfilename.Rmd data/yourdata.txt`.
  5. Then commit this using `git commit` and add some short text describing what your file is.
4. Push to your fork and create a new pull request
  1. From the command line, run `git push --set-upstream origin Your_branch_name` to push your branch to your remote fork
  2. To do a pull request, navigate to your branch on GitHub. Click `Compare & review` or the pull request icon.
  3. Once you have reviewed the changes, click `Create pull request`. Once the request is approved, the file will be automatically sent to a remote server that compiles the document into a html file.

**Comment by smanel** [comment]
This new way to order the step seems to be a good idea . In addition aligning the steps  with the figure is great!

**Comment by zkamvar** [comment]
Should I merge #158?

**Comment by smanel** [comment]
yes.

**Comment by zkamvar** [comment]
Since this is currently a pull request, I will close this issue and let the conversation go on there.
